providers-sdk/v1/inventory/asset_url.go

```diff
@@ -149,10 +149,34 @@ func (a *AssetUrlSchema) Add(branch *AssetUrlBranch) error {
 	}
 
 	branch.setDepth(found.Depth + 1)
-	found.Values[lastKey] = branch
+	found.Values[lastKey] = copyBranch(branch)
```

Comment:
we don't own these, so we copy them. If we don't copy them, then creating an `NewAssetUrlSchema` produces garbage on the second go around as this code modifies it in a way that is not repeatable.